js/src/hooks/useGoogleMCAccount.js

```diff
@@ -19,6 +20,8 @@ import useGoogleAccount from './useGoogleAccount';
  * @property {boolean} isPreconditionReady Whether the precondition of continued connection processing is fulfilled.
  */
 
+const googleMCAccountSelector = 'getGoogleMCAccount';
```

Comment:
Extracting the `googleMCAccountSelector` selector is only partially done, which makes the associated code lengthy and looks like work-in-progress.

https://github.com/woocommerce/google-listings-and-ads/blob/2da86e67698c43ea964440cc86dce62a6778f99e/js/src/hooks/useGoogleMCAccount.js#L52-L55

https://github.com/woocommerce/google-listings-and-ads/blob/2da86e67698c43ea964440cc86dce62a6778f99e/js/src/hooks/useGoogleMCAccount.js#L56

https://github.com/woocommerce/google-listings-and-ads/blob/2da86e67698c43ea964440cc86dce62a6778f99e/js/src/hooks/useGoogleMCAccount.js#L67

https://github.com/woocommerce/google-listings-and-ads/blob/2da86e67698c43ea964440cc86dce62a6778f99e/js/src/hooks/useGoogleMCAccount.js#L69-L70